README.md

```diff
@@ -38,6 +63,18 @@ name.object_id == same_as_name.object_id
   #=> true
 ```
 
+Try printing out the `object_id` for the symbol `:Steven` the same way we did for the strings above. This time we should see the exact same id no matter how many times we run the code. Again, the specific id will differ on your machine. 
+
+```ruby
+puts :Steven.object_id
+puts :Steven.object_id
+puts :Steven.object_id
+
+#=> 1093988
+#=> 1093988
+#=> 1093988
+```
+
 ## Why Use Symbols as Hash Keys?
 
 A hash is like a dictionary—it's a collection that can store data as "values" that are accessed by "keys". The keys are reference points. In other words, we use the keys to access the stored information (the values). Think of a hash like a storage unit. The keys are like the address or location of the storage unit and the values are like the content of the storage unit. Accordingly, we're not interested in manipulating the keys. We don't want to change the address, just use it to look up or access the contents. 
```

Comment:
"The other thing that's nice is since all copies of `:Steven` have the same `object_id`, they use up less memory! `"Steven"` uses a new slot of memory each time you create it. `:Steven` does not